pkg/networkoptions/network_options.go

```diff
@@ -64,6 +64,12 @@ func NetworkOptionFromString(s string) NetworkOption {
 		return Devnet
 	case "Cluster":
 		return Cluster
+	default:
+		if strings.Contains(s, "Devnet") {
+			return Devnet
+		} else if strings.Contains(s, "Cluster") {
+			return Cluster
+		}
```

Comment:
I mean something like 
```
func NetworkOptionFromString(s string) NetworkOption {
	switch {
	case s == "Mainnet":
		return Mainnet
	case s == "Fuji Testnet":
		return Fuji
	case s == "Local Network":
		return Local
	case s == "Etna Devnet":
		return EtnaDevnet
	case s == "Devnet" || strings.Contains(s, "Devnet"):
		return Devnet
	case s == "Cluster" || strings.Contains(s, "Cluster"):
		return Cluster
	default:
		return Undefined
	}
}
```

Reply:
i just built it with this code with no issues 
![image](https://github.com/user-attachments/assets/afa3230f-23fb-400e-95b5-8c9fce65eae1)